app/src/main/java/ar/com/wolox/android/example/network/Users.java

```diff
@@ -0,0 +1,58 @@
+package ar.com.wolox.android.example.network;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * User class for the REST API
+ */
+
+public class Users {
+    @SerializedName("id")
```

Comment:
Since the `@SerializedName` is the same of the attribute name, we could avoid it. But if you think it's better to let it, just do nothing. I only want to let you know that you could avoid it.